packages/google_sign_in/google_sign_in_web/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   google_identity_services_web: ^0.3.0
   google_sign_in_platform_interface: ^2.4.0
   http: ">=0.13.0 <2.0.0"
-  web: ">=0.3.0 <0.5.0"
+  web: ">=0.3.0 <0.6.0" # because google_identity_services
```

Comment:
Is this comment explaining the lower limit or the upper limit? More details would help.

Reply:
I *think* this is because `google_identity_services` has not been upgraded yet...

Reply:
The comment explains the upper limit, this is the package I'll immediately update after landing this PR, so the upper limit will live there only for one small publish.